packages/app/src/main.tsx

```diff
@@ -1,11 +1,22 @@
 import '@fontsource/source-code-pro';
+import type { ReactNode } from 'react';
 import { createRoot } from 'react-dom/client';
-import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter, HashRouter } from 'react-router-dom';
 
 import { App } from './App';
+import { IS_CRX } from './config';
+
+const RouterProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
We have another component that needs to make some routing logic that's `TestWrapper`, now with this another routing logic here, we can start to have some confusion around this. Maybe we could have a `Core/Router` component that handles all this inside of it and could be used in all parts of the application: as a wrapper for tests on jest, as a router for CRX and the web.

Reply:
I was looking at the TestWrapper is used directly on the tests, don't see how to combine it, or if was the idea. But I moved the router provider to inside the system/core